configs/edison/job_script.edison.bash

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash -l
+
+# comment out if using debug queue
+#SBATCH --partition=regular
+# comment in to get premium queue
+##SBATCH --qos=premium
+# comment in to get the debug queue
+##SBATCH --partition=debug
+# change number of nodes to change the number of parallel tasks
+#SBATCH --nodes=10
```

Comment:
should the number of nodes be the same as the number of tasks, or it doesn't matter?